backend/tests/use_cases/test_user_interactor.py

```diff
@@ -1,11 +1,9 @@
 import pytest
 from unittest.mock import MagicMock
 from app.repositories.user_repository import UserRepository
-from app.use_cases.user_interactor import (
-    register_user_interactor,
-    login_user_interactor,
-    change_password_interactor
-)
+from app.use_cases.register_user_interactor import RegisterUserInteractor
```

Comment:
Does this follow the dependency rule?

Reply:
Yes, my commit follows the Dependency Rule. The interactors (RegisterUserInteractor, LoginUserInteractor, ChangePasswordInteractor) depend on abstractions (the UserRepository) rather than concrete implementations. This allows for easier testing and maintains separation of concerns, with the interactor layer remaining independent of the repository and database layers.